src/tests/ftest/util/server_utils.py

```diff
@@ -290,3 +271,31 @@ def kill_server(hosts):
     for host in hosts:
         subprocess.call("ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)),
                         shell=True)
+
+def storage_prepare(hosts):
+    """
+    Prepare the storage on servers using the DAOS server's yaml settings file.
+
+    Args:
+        hosts (str): a string of comma-separated host names
+    """
+    try:
+        cmd = ("sudo /usr/bin/daos_server storage prep-nvme "
```

Comment:
Landing this along with the `full_regression` tag added in the test above is going to cause this test to break (once the work to enable the `full_regression` testing has landed) until avocado testing has been migrated to testing with RPMs.

How will we prevent that (i.e. landing prematurely)?

Given that this needs to wait to land (or be changed so that it doesn't break things when the `full_regression` work is landed), is it prudent to extract the clustershell bits into a separate patch and get that landed to prevent any more investment in re-inventing clustershell?

Reply:
I am trying to get the clustershell in another PR#641.

Reply:
> Landing this along with the `full_regression` tag added in the test above is going to cause this test to break (once the work to enable the `full_regression` testing has landed) until avocado testing has been migrated to testing with RPMs.
> 
> How will we prevent that (i.e. landing prematurely)?
> 

https://github.com/sanemat/do-not-merge-wip-for-github. Right now merge button is disabled as it does not have enough review but I added the text so hopefully merge button will still be disabled unless we remove the text from PR comment .

Reply:
That looks like a browser extension and so will only work as long as all gatekeepers use chrome and have that extension installed.  That's probably not a reasonable requirement of gatekeepers.

Reply:
@brianjmurrell I agree can you please open JIRA task/ticket for it and add the additional requirement you want from clustershell. I will start working once I am done with current task.

Reply:
[DAOS-2750](https://jira.hpdd.intel.com/browse/DAOS-2750)